### PR TITLE
call positionTether on show() to ensure correct tooltip placement

### PR DESCRIPTION
**Pull request by a15n**
addresses this issue https://github.com/sir-dunxalot/ember-tooltips/issues/75

### CODE REVIEW

**Comment by sir-dunxalot** [comment]
Ah, gotcha, great.

File: addon/components/tooltip-on-element.js [hunk 1]
```diff
@@ -381,6 +381,7 @@ export default EmberTetherComponent.extend({
   }),
 
   show() {
+    this.positionTether();
```
Comment (sir-dunxalot) [inline addon/components/tooltip-on-element.js]: Any chance you can add an inline comment to say this is here to fix #75?

**Comment by sir-dunxalot** [comment]
One comment; otherwise this looks good.

**Comment by a15n** [comment]
I added the note. Thanks for the quick turnaround on this.

**Comment by sir-dunxalot** [comment]
Of course! This is now on master. I still need to fix the test suite for `ember-beta` before making the next release.

**Review by Misshigh77 — COMMENTED** [review]
> #455

**Pull request merged**